packages/analytics/analytics-chart/src/utils/formatUnit.ts

```diff
@@ -0,0 +1,102 @@
+import composables from '../composables'
+import { numberFormatter } from './constants'
+
+export const formatBytes = (value: number, isBytes1024?: boolean): string => {
```

Comment:
We're already using https://www.npmjs.com/package/pretty-bytes to format byte display. 

Did you want to discuss further why we should use something custom/in-house instead of that package? 

We should at least be consistent and either use pretty-bytes everywhere, or update the area where pretty-bytes is currently used to use your new function